notebooks/MUS2019_CTF.ipynb

```diff
@@ -0,0 +1,1120 @@
+        "ts_results = ctf.explore(\n",
+        "    'data_type:\"windows:registry:key_value\" AND key_path:\"HKEY_LOCAL_MACHINE\\\\\\System\\\\\\Select\"', \n",
+        "    return_fields='datetime,timestamp_desc,data_type,message', \n",
+        "    as_pandas=True)\n",
+        "ts_results[['datetime','timestamp_desc','data_type','message']]"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "FoqCvGtlFnZ1",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "From the message, the Current control set is 1."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "3QuYQux5FtlO",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### Q: What was the timezone offset at the time of imaging? and What is the timezone of the Desktop"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "sX5xYElTHxOh",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "I'm combining these, since the answer is in the same query:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "f8CE4NYEFzWu",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "ts_results = ctf.explore(\n",
+        "    'data_type:\"windows:registry:key_value\" AND key_path:\"HKEY_LOCAL_MACHINE\\\\\\System\\\\\\ControlSet001\\\\\\Control\\\\\\TimeZoneInformation\"', \n",
+        "    return_fields='datetime,timestamp_desc,data_type,message', \n",
+        "    as_pandas=True)\n",
+        "ts_results[['datetime','timestamp_desc','data_type','message']]"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "HJDjeFc0bzu4",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The message is really long; let's pull it out:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "s8UMrq0tb3u6",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "set(ts_results.message)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "NAloEuihHke2",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The name of the Timezone is in the message string, as is the ActiveTimeBias, which we can use to get the UTC offset:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "5R57xnzQGzEB",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# The ActiveTimeBias is in minutes, so divide by -60 (I don't know why it's stored negative): \n",
+        "420 / -60"
```

Comment:
although this is a bit more complicated text... should we do this more "programmatically" ? food for thought... something like:

```
time_string = list(set(ts_results.message))[0]
_, _, data_string = time_string.partition(']')
data = {}
items = data_string.split(':')
key = items[0].strip()

for line in items[1:-1]:
  words = line.split()
  data[key] = ' '.join(words[:-1])
  key = words[-1]
data[key] = items[-1]

print('The UTC offset is: {0:d}'.format(int(data.get('ActiveTimeBias', '0')) / int(data.get('DaylightBias', '1'))))
```

Reply:
I'd rather keep it simpler for the target audience. The notebook is supposed to show a "live" investigation, and writing that much code for a small one-time operation seems like overkill to me.

I also think it'd be good to take another look at how plaso parses these out (separate entries per key, rather than all into one string).

Reply:
ok... agree... but maybe change the text a bit to show or talk about where those numbers come from...